src/wrappers/themis/wasm/wasmthemis.mk

```diff
@@ -24,14 +24,19 @@ WASM_PRE_JS  = $(abspath $(WASM_PATH)/emscripten/pre.js)
 
 WASM_PACKAGE = $(BIN_PATH)/wasm-themis.tgz
 
-$(BIN_PATH)/libthemis.js: LDFLAGS += -s EXTRA_EXPORTED_RUNTIME_METHODS=@$(WASM_RUNTIME)
+$(BIN_PATH)/libthemis.js: LDFLAGS += -s EXPORTED_RUNTIME_METHODS=@$(WASM_RUNTIME)
```

Comment:
why has the changed variable's name? who depends on it?

Reply:
> why has the changed variable's name?

[Emscripten changelog](https://emscripten.org/docs/introducing_emscripten/release_notes.html) quoted in commit message:

```
2.0.18: 04/23/2021
------------------

- `EXTRA_EXPORTED_RUNTIME_METHODS` is deprecated in favor of just using
  `EXPORTED_RUNTIME_METHODS`.
```

> who depends on it?

That would be the linker. This tells what Emscripten runtime methods to retain so that WasmThemis can make use of them.